back/engines/commercial/analysis/spec/lib/nlp_cloud_helpers.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Analysis::NLPCloudHelpers do
+  describe 'LOCALE_MAPPING' do
+    it 'contains an entry for all our locales' do
+      expect(described_class::LOCALE_MAPPING.keys).to match_array CL2_SUPPORTED_LOCALES.map(&:to_s)
+    end
+  end
```

Comment:
Avoid the 90+ character line and is coherent with the describe:
```suggestion
  describe 'LOCALE_MAPPING' do
    subject(:mapping) { described_class::LOCALE_MAPPING }

    it 'contains an entry for all our locales' do
      expect(mapping.keys).to match_array CL2_SUPPORTED_LOCALES.map(&:to_s)
    end
  end
```